corehq/ex-submodules/dimagi/utils/django/email.py

```diff
@@ -51,13 +51,21 @@ def mark_local_bounced_email(bounced_addresses, message_id):
             )
 
 
-def get_valid_recipients(recipients):
+def get_valid_recipients(recipients, domain=None):
     """
     This filters out any emails that have reported hard bounces or complaints to
     Amazon SES
     :param recipients: list of recipient emails
     :return: list of recipient emails not marked as bounced
     """
+    from corehq.toggles import BLOCKED_DOMAIN_EMAIL_SENDERS
+    if domain and BLOCKED_DOMAIN_EMAIL_SENDERS.enabled(domain):
```

Comment:
@biyeun I know this is from like a year and a half ago, but I just noticed that this perhaps accidentally doesn't block invitations, because they don't specify the domain when sending the email

https://github.com/dimagi/commcare-hq/blob/8504a486dd421fc4731d63a1e3632e4080f9b601/corehq/apps/users/models.py#L2625-L2629

Not sure whether that's desirable or not, but it seemed questionable, so wanted to run it by you.